cx-portal/src/components/pages/AboutPage/index.tsx

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (c) 2021, 2023 BMW Group AG
+ * Copyright (c) 2021, 2023 Mercedes-Benz Group AG and BMW Group AG
```

Comment:
It's better to entirely remove the company specific line, and just leave the one for EF.